Sources/Library/Library.swift

```diff
@@ -0,0 +1,40 @@
+import Foundation
+
+/// An example of a struct with only `Sendable` properties.
+///
+/// This type is implicitly-`Sendable` within its defining module.
```

Comment:
Ah, this isn't true. There's no such thing as internal-only conformances, so this struct isn't implicitly `Sendable` at all. That said, inferring `Sendable` only within the module would be an interesting future direction!

Reply:
Ohhh my mistake! I did not realize that making them public removes the implict conformance. Shoot, I think I have some written content to update too now...